cli/test.yaml

```diff
@@ -3759,3 +3759,26 @@
       "foo": 10
     }
   error: 'open testdata/6.json:'
+
+- name: 'single variable'
+  args:
+    - '-amul=10'
```

Comment:
Ah, can I remove the short flag? `-a` in jq has different meaning and might be confusing.

Reply:
I've removed the `-a` flag. `--arg` is still provided by jq and has a different meaning there as well (the `--arg` in gojq is like a combination of jq's `--arg` and `--argjson`, defaulting to argjson if it works and falling back to arg if it doesn't). That can be renamed as well, or I can just remove the cli options entirely and implement those later.

Reply:
I got it.